src/services/projectService.ts

```diff
@@ -18,8 +18,8 @@ class ProjectService {
    * @param id - ID of the Project
    * @returns Project
    */
-  public getProject(id: string) {
-    return pncClient.getHttpClient().get(`${this.path}/${id}`);
+  public getProject(requestConfig = {}, id: string) {
```

Comment:
let's swap arguments to: `getProject(id: string, requestConfig = {})`, and WDYT about using named arguments?